applications/nrf5340_audio/src/bluetooth/Kconfig

```
@@ -20,6 +20,38 @@ config BLE_ACL_SUP_TIMEOUT
 	int "Bluetooth LE Supervision Timeout (x*10ms)"
 	default 400
 
+config BLE_ACL_PER_ADV_INT_MIN
+	hex "Minimum periodic adv interval"
+	range 0x0018 0x03C0
+	default 0x0078
+	help
+	  Minimum value for the interval of periodic advertisements
+	  Actual value in ms is x1.25
+
+config BLE_ACL_PER_ADV_INT_MAX
+	hex "Maximum periodic adv interval"
+	range 0x0018 0x03C0
+	default 0x00A0
+	help
+	  Minimum value for the interval of periodic advertisements
+	  Actual value in ms is x1.25
+
+config BLE_ACL_EXT_ADV_INT_MIN
+	hex "Minimum extended adv interval"
+	range 0x0030 0x0780
+	default 0x00A0
+	help
+	  Minimum value for the interval of extended advertisements
+	  Actual value in ms is x0.625
+
+config BLE_ACL_EXT_ADV_INT_MAX
+	hex "Maximum extended adv interval"
+	range 0x0030 0x0780
+	default 0x00F0
+	help
+	  Maximum value for the interval of extended advertisements
+	  Actual value in ms is x0.625
+
 config BLE_LE_POWER_CONTROL_ENABLED
 	bool "Enable LE power control feature"
```

```suggestion
	bool "Enable Bluetooth LE power control feature"
```
LE, BLE, etc. are banned by BT SIG terminology guidelines... 😢